### ISSUE TITLE
[1.5 beta] Masquer un répertoire vide

### ISSUE THREAD

**Issue by cobalt74** [barbogogo/leedReader#72]
Hello @barbogogo super la 1.5 :+1: 

Petit bug qui subsiste  et qui était déjà présent sur la version d'avant.

quand je paramètre pour masquer les répertoires vides, quand je retourne sur l'accueil, j'ai ce message :

Leed-API : v0.10

```
APIConnection Erreur #3
No Value for Folders
```

**Comment by barbogogo** [comment]
Salut @cobalt74,
Merci :smile:

Petite question : est-ce que tu as ce problème lorsque tu lis un article, tous les articles d'un flux ou tous les articles de tous les flux ?

**Comment by cobalt74** [comment]
@barbogogo tous les flux

**Comment by cobalt74** [comment]
j'ai tjrs le problème en 1.5.1 béta.

**Comment by barbogogo** [comment]
Oui, je n'ai pas encore réussi à reproduire le problème, mais je suis dessus.
